src/core/builders/vite/plugins/multipageMove.ts

```diff
@@ -67,6 +67,30 @@ export function multipageMove(
         delete bundle[oldBundlePath];
         bundle[newBundlePath] = renamedChunk;
       }
+
+      // Remove directories that were created
+      // TODO: Optimize and only delete old path directories
+      removeEmptyDirs(config.outDir);
```

Comment:
TODO. We should be able to track all the directories we moved files out of, and delete those and their parents if they're empty.